addons/mail/static/src/core/common/composer.js

```diff
@@ -803,13 +811,29 @@ export class Composer extends Component {
         }
     }
 
-    restoreContent() {
+    async restoreContent() {
```

Comment:
The PR should also allow restoring `specialMentions` (`@everyone` in particular).